### PR TITLE
Add numastat collector. Add tests

### PR DESCRIPTION
**Pull request by dichiarafrancesco**

### CODE REVIEW

**Comment by dichiarafrancesco** [comment]
`{
"type":"CUMCOUNTER",
"name":"numastat.local_node",
"value":7545953589.0,
"dimensions":{
   "node":"node0",
   "prefix":"servers",
   "collector":"numastat",
   "collectorCanonicalName":"NumastatCollector"}
}`

**Comment by dichiarafrancesco** [comment]
This is a kind of output metrics:

{
"type":"CUMCOUNTER",
"name":"numastat.local_node",
"value":7545953589.0,
"dimensions":{
   "node":"node0",
   "prefix":"servers",
   "collector":"numastat",
   "collectorCanonicalName":"NumastatCollector"}
}

File: src/diamond/collectors/numastat/numastat.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        })
+        return config_help
+
+    def get_default_config(self):
+        """
+        Returns the default collector settings
+        """
+        config = super(NumastatCollector, self).get_default_config()
+        config.update({
+            'path': 'numastat'
```
Comment (gnufied) [inline src/diamond/collectors/numastat/numastat.py]: We don't seem to be using this configuration param anywhere in code.

**Comment by gnufied** [comment]
Fix & Ship!!

File: src/diamond/collectors/numastat/numastat.py (hunk 1 above)
Comment (dichiarafrancesco) [inline src/diamond/collectors/numastat/numastat.py]: `path` param is always present in diamond collectors. i think it's better to keep uniformity.

Comment (gnufied) [inline src/diamond/collectors/numastat/numastat.py]: ah okay. ignore my comment then. :D

**Pull request merged**